processor/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'java'
+
+dependencies {
+    compile project(":annotations")
+    compile 'com.squareup:javapoet:1.0.0'
```

Comment:
All our dependencies are declared in the root `build.gradle` and referenced with variables. This way we prevent version skew for multiple modules using the same dependency. Can you update all of these in that style?